src/cgutils.cpp

```diff
@@ -2156,25 +2156,10 @@ static Value *emit_allocobj(jl_codectx_t &ctx, size_t static_size, Value *jt)
 {
     JL_FEAT_REQUIRE(ctx, dynamic_alloc);
     JL_FEAT_REQUIRE(ctx, runtime);
```

Comment:
Should probably postpone these checks to the actual lowering (but we don't have a `cgctx` there).